README.md

```diff
@@ -1 +1,15 @@
-# firebase
+# Firebase Chat - A mentoring application
+
+Simple web app chat using firebase features.
+
+## Features to implement
+
+- [ ] Users `Authentication` via email, Google or Facebook, and maybe a phone number.
+- [ ] Profiles stored via `Cloud Firestore`
+- [ ] Connecting to `Realtime Database`
+- [ ] Adding counters, creation of rooms, and others via `Cloud Functions`
+- [ ] `Cloud Storage` maybe users profile avatars :question:
```

Comment:
nie jestem przekonany czy używanie dwóch baz nie będzie overkillem, ale to chyba zdecydujemy po researchu. Przeniesienie userów do zwykłej bazy nie będzie problemem w razie czego

Reply:
Dla mnie ogolnie wszystko spoko, ta druga baze, mozemy wykorzystac jedynie zeby ja pokazac ale bez niej te profile tez moglyby byc live.

Reply:
FYI https://firebase.google.com/docs/storage/
Tak oczywiście, wszystko można by zapisać w jednej bazie. Sugeruje taki podział funkcjonalności, jedynie po to by pokryć jak największą ich ilość funkcjonalnościami dostępnymi z Firebase'a.